### PR TITLE
Create ui-c8y-1019-23-1-tfa-strategy-updated-by-tfa-endpoint-6146-graft-release-cd-6211.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-52862](https://cumulocity.atlassian.net/browse/MTM-52862)
Original PR: [6211](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6211)

### CODE REVIEW

File: content/change-logs/platform-services/ui-c8y-1019-23-1-tfa-strategy-updated-by-tfa-endpoint-6146-graft-release-cd-6211.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: tfa strategy updated by tfa endpoint (#6146) [GRAFT][release/cd] (#6211)
```
Comment (gen-App-Enablement) [inline content/change-logs/platform-services/ui-c8y-1019-23-1-tfa-strategy-updated-by-tfa-endpoint-6146-graft-release-cd-6211.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Update TFA strategy via TFA endpoint
```

File: content/change-logs/platform-services/ui-c8y-1019-23-1-tfa-strategy-updated-by-tfa-endpoint-6146-graft-release-cd-6211.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-0UgqXH1Ys
+    label: Administration
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-52862
+version: 1019.23.1
+---
+tfa strategy updated by tfa endpoint (#6146) [GRAFT][release/cd] (#6211)
```
Comment (gen-App-Enablement) [inline content/change-logs/platform-services/ui-c8y-1019-23-1-tfa-strategy-updated-by-tfa-endpoint-6146-graft-release-cd-6211.md:17]: The Claude AI suggested to use the following body:
```suggestion
In an effort to enhance the security and flexibility of the two-factor authentication (TFA) process, the TFA strategy can now be updated using the dedicated TFA endpoint. This change allows users to modify their TFA settings more easily and efficiently, without needing to navigate through multiple steps or menus. The updated functionality streamlines the TFA configuration process, providing a more user-friendly experience while maintaining the robust security measures that TFA offers.
```

**Review by pink-mongoose — COMMENTED** [review]
I really like the Claude AI generated text, is that what will be used?

**Comment by janhommes** [comment]
> I really like the Claude AI generated text, is that what will be used?

Yes, simply commit the suggestions. You can even edit the comments if you want to align something.

**Review by pink-mongoose — APPROVED** [review]
I committed the AI proposals but would like the following change:

Remove the 'In an effort ...' so the release note begins, 'To enhance the security ...'

**Comment by BeateRixen** [comment]
@kalavathig1 @sarangovindarajan Please follow-up here. In general, please always follow-up on automatically created PRs with label "authentication".

**Review by mmed-softwareag — APPROVED** [review]

File: content/change-logs/platform-services/ui-c8y-1019-23-1-tfa-strategy-updated-by-tfa-endpoint-6146-graft-release-cd-6211.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-0UgqXH1Ys
+    label: Administration
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-52862
+version: 1019.23.1
+---
+When changing the TFA strategy, the application must update the strategy for all users. This effect is not implemented if the strategy update occurs through changing tenant options. Changing the endpoint ensures that after changing the strategy on the configuration page, the strategy for all users will be updated.
```
Comment (BeateRixen) [inline content/change-logs/platform-services/ui-c8y-1019-23-1-tfa-strategy-updated-by-tfa-endpoint-6146-graft-release-cd-6211.md:17]:
```suggestion
When changing the TFA method for a tenant, the application must update the method for all users. This did not happen if the method was changed through the tenant options. Changing the method through the TFA endpoint now ensures that after changing the method on the configuration page, the method will be updated for all users.
```

File: content/change-logs/platform-services/ui-c8y-1019-23-1-tfa-strategy-updated-by-tfa-endpoint-6146-graft-release-cd-6211.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: The update of the TFA strategy takes place through the TFA endpoint.
```
Comment (BeateRixen) [inline content/change-logs/platform-services/ui-c8y-1019-23-1-tfa-strategy-updated-by-tfa-endpoint-6146-graft-release-cd-6211.md:3]:
```suggestion
title: Changing the TFA method for a tenant now updates it for all users
```

File: content/change-logs/platform-services/ui-c8y-1019-23-1-tfa-strategy-updated-by-tfa-endpoint-6146-graft-release-cd-6211.md (hunk 3 above)
Comment (pink-mongoose) [reply content/change-logs/platform-services/ui-c8y-1019-23-1-tfa-strategy-updated-by-tfa-endpoint-6146-graft-release-cd-6211.md:17]: @BeateRixen I like your suggested change much more

File: content/change-logs/platform-services/ui-c8y-1019-23-1-tfa-strategy-updated-by-tfa-endpoint-6146-graft-release-cd-6211.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: TFA strategy is now updated through the TFA endpoint
+product_area: Platform services
+change_type:
+  - value: change-VSkj2iV9m
+    label: Fix
+component:
+  - value: component-0UgqXH1Ys
+    label: Administration
```
Comment (BeateRixen) [inline content/change-logs/platform-services/ui-c8y-1019-23-1-tfa-strategy-updated-by-tfa-endpoint-6146-graft-release-cd-6211.md:10]:
```suggestion
    label: Authentication
```

File: content/change-logs/platform-services/ui-c8y-1019-23-1-tfa-strategy-updated-by-tfa-endpoint-6146-graft-release-cd-6211.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: TFA strategy is now updated through the TFA endpoint
+product_area: Platform services
+change_type:
+  - value: change-VSkj2iV9m
+    label: Fix
+component:
+  - value: component-0UgqXH1Ys
```
Comment (BeateRixen) [inline content/change-logs/platform-services/ui-c8y-1019-23-1-tfa-strategy-updated-by-tfa-endpoint-6146-graft-release-cd-6211.md:9]:
```suggestion
  - value: q3kclF6pO 
```

**Review by kalavathig1 — APPROVED** [review]

**Pull request merged**